### ISSUE TITLE
CARMA UI - Remote Start and Starting RVIZ automatically

### ISSUE THREAD

**Issue by maefromm** [usdot-fhwa-stol/CARMAPlatform#461]
### Types of Issue

- [x] Anomaly report (something appears to not work correctly)
- [ ] Enhancement request (describe the enhancement being requested)
- [ ] Other (please ensure the description clarifies why the issue doesn’t fall into either of the above categories)

### Descriptive summary
CARMA 2.9 UI had a checkbox for Remote Start that will start the CARMAPlatform automatically without any other action needed. 

With CARMA3, multiple actions need to be taken to allow the CARMAPlatform to fully start up and shutdown. For startup, 
1) Double click on CARMA3 Desktop icon, which should automatically open RVIZ. 
2) Manually localize using RVIZ - This is logged in a separate issue #384 
3) Double click on the CARMA3 Web icon to open the CARMA Web UI. Uncheck the Remote Start checkbox since it's no longer needed or may not work. 
4) When operator is done, click the logout button on the UI. This should shutdown all of the CARMAPlatform nodes, however this does not happen at this time. This is logged in a separate issue #460. Thus, this means the operator has to manually terminate the terminal(s) to shutdown the application, and close out RVIZ. And close out the web interface. 

As discussed with @Supraluminal we need to discuss the design and effort to do this. 

### Expected behavior
1) With one press of a button, expect that CARMA3 Web UI's Remote Start checkbox allows the operator to start the application and open RVIZ (with correct profile) to verify localization is good. 

2) With one press of a button, expect that CARMA3 Web UI's logout allows the operator to stop the applicaiton and RVIZ automatically.

### Software version this applies to

3.1.0

### Actual behavior
Explained above. 

### Steps to reproduce the behavior
Explained above. 

### Related work
Link to related tickets or prior related work here.
Related issues that maybe related and already known and considered separate from this issue: 
https://github.com/usdot-fhwa-stol/CARMAPlatform/issues/384
https://github.com/usdot-fhwa-stol/CARMAPlatform/issues/460
https://github.com/usdot-fhwa-stol/CARMAPlatform/issues/145
https://github.com/usdot-fhwa-stol/CARMAPlatform/issues/453

**Comment by msmcconnell** [comment]
Rviz functionality is blocked by
https://github.com/usdot-fhwa-stol/carma-platform/issues/854
https://github.com/usdot-fhwa-stol/carma-platform/issues/733
